lib/plugins/aws/package/compile/events/apiGateway/lib/hack/updateStage.js

```diff
@@ -208,19 +210,43 @@ function handleLogs() {
       }
     }
 
-    const destinationArn = {
-      op: 'replace',
-      path: '/accessLogSettings/destinationArn',
-      value: `arn:aws:logs:${region}:${this.accountId}:log-group:${logGroupName}`,
-    };
-    const format = {
+    let enableAccessLogs = logs.enableAccessLogging;
+    if (enableAccessLogs === undefined) {
+      enableAccessLogs = true;
+    }
```

Comment:
More bulletproof would be to check for _no value_ (neither `null` or `undefined`) as:

```javascript
let accessLogs = logs.accessLogs == null ? true : logs.accessLogs;
```

In not distant feature we should have [nullish coalescing operator](https://2ality.com/2019/08/nullish-coalescing.html) syntax, that will allow to write same as:

```javascript
let accessLogs = logs.accessLogs ?? true;
```

Reply:
Done.